requirements.txt

```diff
@@ -18,11 +18,10 @@ django-sortedm2m==1.5.0
 django-bower==5.2.0
 invoke==1.2.0
 requests==2.21.0
-whitenoise==3.3.1
+whitenoise==4.1.2
```

Comment:
@Emmanuel-Dominic any particular reason for changing this version to a higher version?

Reply:
Feedback worked on, i have et it back to default the previous version.